src/main/java/com/icthh/xm/ms/entity/domain/AttachmentStoreType.java

```diff
@@ -0,0 +1,8 @@
+package com.icthh.xm.ms.entity.domain;
+
+/**
+ * Enum for attachment storage type
+ */
+public enum AttachmentStoreType {
+    DB, AWS;
```

Comment:
consider rename AWS to S3. Because we are using S3 and AWS is just one platform implementing S3 storage type.